### DISCUSSION TITLE
Unread notification from removed user/repo

### DISCUSSION THREAD

**Discussion by bato3** [community/community#33632] [Other Feature Feedback, Questions, & Ideas] [answered]
I have notyfication, that I have unread nottification. Also I have "1" on "Inbox". But when I choose it (https://github.com/notifications?query=is%3Aunread) list is empty.

I'm thinking, that is from removed user, because in "repositores" section I have `aidoostech/AiDOOS-11`, but there is no user `aidoostech` on github

**Comment by radar-blip** [comment, marked as answer]
That is a 'ghost notification',  a glitch that happens every once in a while. Not a big deal.
To get rid of it, mark all notifications as 'Read' and it should go away. Works for me.

I guess I should add: whether you can see them or not, 'select all' notifications, and mark as read.
Also, other users may have a different method and/or fix. What works for me might not work for you.

**Reply by bato3** [reply]
Thanks for help. "Ghost notification" was subject, that I need. 
From thread: https://github.com/community/community/discussions/6874#discussioncomment-1872442

1. move threads to Inbox (more than pagination limit)
2. Select All, and use link to select threads from another pages

**Reply by radar-blip** [reply]
You're welcome! Glad it worked for you. 
Now you know how to do it next time. 😀
